config/markets.ts

```diff
@@ -320,6 +320,52 @@ const config: {
   [network: string]: MarketConfig[];
 } = {
   arbitrum: [
+    {
+      tokens: { indexToken: "APE", longToken: "APE", shortToken: "USDC" },
+      virtualTokenIdForIndexToken: hashString("PERP:APE/USD"),
+      virtualMarketId: hashString("SPOT:APE/USD"),
+
+      ...baseMarketConfig,
+
+      positionImpactExponentFactor: decimalToFloat(22, 1), // 2.2
+      positivePositionImpactFactor: decimalToFloat(25, 11), // 0.05% for ~90,000 USD of imbalance
+      negativePositionImpactFactor: decimalToFloat(5, 10), // 0.05% for ~45,000 USD of imbalance
+
+      positiveSwapImpactFactor: decimalToFloat(15, 9), // 1.5e-8
+      negativeSwapImpactFactor: decimalToFloat(3, 8),
+
+      borrowingFactorForLongs: decimalToFloat(160, 10), // 1.60E-08, ~50% if 100% utilized
+      borrowingFactorForShorts: decimalToFloat(160, 10), // 1.60E-08, ~50% if 100% utilized
+
+      fundingIncreaseFactorPerSecond: decimalToFloat(116, 14), // 0.00000000000116, at least 3.5 hours to reach max funding
```

Comment:
Chaos suggested different funding increase factor 1.74E-12, is it expected to have 1.16e-12? 

checking here https://www.notion.so/chaos-labs/New-Asset-Listing-APE-4e70bdbaf75c4ca084eca86024aa7759